app/controllers/application_controller.rb

```diff
@@ -12,29 +12,29 @@ class RbacError < StandardError; end
   around_action :with_current_request
 
   rescue_from ActiveRecord::RecordNotFound do |exception|
```

Comment:
I'm ok with these `rescue_from`'s for now since they were already here, but we should probably move these to an exception mapping class and let the common gem handle it like we do for catalog/approval.

I'll make an issue for it.